### PR TITLE
v13

### PR DESCRIPTION
**Pull request by WilliamDenniss**

### CODE REVIEW

**Review by selfissued — APPROVED** [review]
LGTM

**Comment by WilliamDenniss** [comment]
Thank you Mike. I added some more changes to address outstanding DISCUSS comments, PTAL.

File: draft-ietf-oauth-device-flow.xml [hunk 1]
```diff
@@ -86,15 +86,16 @@
...
+        This OAuth 2.0 authorization flow is designed for devices
+        that either lack a browser to perform a user-agent based OAuth
+        flow, or are input-constrained to the extent that requiring the user to
+        input a lot of text (like their credentials to authenticate with the
+        authorization server) is impractical.
+        It enables OAuth clients on such devices (like smart TVs,
+        media consoles, digital picture frames, and
+        printers) to obtain user authorization to access protected
+        resources without using an on-device user-agent, provided
+        that they have an internet connection.
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: internet -> Internet

File: draft-ietf-oauth-device-flow.xml [hunk 2]
```diff
@@ -106,7 +107,7 @@
       <t>
         This <xref target="RFC6749">OAuth 2.0</xref> protocol flow for browserless and input-constrained
         devices, often referred to as the device flow, enables OAuth clients to
-        request user authorization from devices that have an internet
+        request user authorization from applications on devices that have an internet
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: internet -> Internet

File: draft-ietf-oauth-device-flow.xml [hunk 3]
```diff
@@ -121,14 +122,30 @@
         <xref target="RFC8252">OAuth 2.0 for Native Apps</xref>.
       </t>
 
+        <t>The operating requirements to be able to use this authorization
+        flow are:
+	    <list style="format (%d)"> 
+        <t>
+          The device is already connected to the internet.
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: internet -> Internet

File: draft-ietf-oauth-device-flow.xml [hunk 4]
```diff
@@ -121,14 +122,30 @@
...
-        The only requirements to use this flow are that the device is connected
-        to the Internet, and able to make outbound HTTPS requests, be able to
-        display or otherwise communicate a URI and code sequence to the user,
-        and that the user has a secondary device (e.g., personal computer or
-        smartphone) from which to process the request. There is no requirement
-        for two-way communication between the OAuth client and the user-agent,
-        enabling a broad range of use-cases.
+        As the device flow does not require two-way communication
+        between the OAuth client and the user-agent (unlike other
+        OAuth 2 flows), it supports several use-cases that cannot be
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: use-cases -> use cases

File: draft-ietf-oauth-device-flow.xml [hunk 5]
```diff
@@ -137,7 +154,18 @@
         to the authorization server to approve the access request.  Since the
         client cannot receive incoming requests, it polls the authorization
         server repeatedly until the end user completes the approval process.
-      </t> 
+      </t>
+
+      <t>
+        The device typically chooses the set of authorization servers to
+        support (i.e. its own authorization server, or those by providers
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: There's always a comma required after "i.e." (and after "e.g.").

File: draft-ietf-oauth-device-flow.xml [hunk 6]
```diff
@@ -421,16 +450,23 @@
     <spanx style="verb">verification_uri</spanx> and continues with them
     supplying the <spanx style="verb">user_code</spanx> at some stage during
     the interaction. Other than that, the exact sequence and implementation of
-    the user interaction is up to the authorization server and is out of scope
-    of this specification.
+    the user interaction is up to the authorization server, for example
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: Add a comma after "for example".

File: draft-ietf-oauth-device-flow.xml [hunk 7]
```diff
@@ -421,16 +450,23 @@
     <spanx style="verb">verification_uri</spanx> and continues with them
     supplying the <spanx style="verb">user_code</spanx> at some stage during
     the interaction. Other than that, the exact sequence and implementation of
-    the user interaction is up to the authorization server and is out of scope
-    of this specification.
+    the user interaction is up to the authorization server, for example 
+    the authorization server may enable new users to sign-up for an account
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: sign-up -> sign up

**Review by selfissued — APPROVED** [review]
After fixing the nits identified in my review, I'm good with merging this.

**Comment by WilliamDenniss** [comment]
> Please put periods at the end of the sentences in the changelog.

That's done.

**Pull request merged**